src/test/java/co/rsk/bitcoinj/script/ErpFederationRedeemScriptParserTest.java

```diff
@@ -43,386 +41,6 @@ public void extractStandardRedeemScriptChunks_fromStandardRedeemScript_fail() {
         ErpFederationRedeemScriptParser.extractStandardRedeemScriptChunks(standardRedeemScript.getChunks());
     }
 
-    @Test
```

Comment:
All the tests removed will be analyzed and moved to a new class called `RedeemScriptUtilsTest.java`, to cover the `createErpRedeemScrip` method with tests.

Reply:
Would it be worth it to include that in this PR?

Reply:
I'd rather create a new PR with these changes and this PR keep it to get rid of methods and tests.

Reply:
I don't think we should move this to a RedeemScriptUtilsTest class, since RedeemScriptUtils is a utils class in the tests package.

Rather these tests should actually be in rskj, where the new ErpRedeemScriptBuilder is used. Make sure the builder in rskj behaves the same as the one here.

Perhaps we should tackle that first before removing this code?

Reply:
@marcos-iov Good catch related to the `ErpRedeemScriptBuilder` in `RSKj`. Concerning the code in the `RedeemScriptUtils`, it is not being tested by any test. Shouldn't this `Utils` class have tests?